src/components/WhatWeDoSection/WhatWeDoSection.jsx

```diff
@@ -7,8 +7,10 @@ import { Heading } from '../../legos/typography/Heading';
 export const WhatWeDoSection = ({ withBackground }) => (
   <Box
```

Comment:
I guess the background and top padding are wrong. Kindly compare:

* Design
![image](https://user-images.githubusercontent.com/12502321/84645378-247d9d80-af09-11ea-9bb9-1f8115309b6c.png)

* Markup
![image](https://user-images.githubusercontent.com/12502321/84645353-1891db80-af09-11ea-968c-a1b0a2d47610.png)